iac/cdk_diff.sh

```
@@ -0,0 +1,37 @@
+function diff(){
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)